Testscripts/Linux/customKernelInstall.sh

```diff
@@ -180,6 +180,40 @@ function Build_Kernel() {
 }
 
 function InstallKernel() {
+    if [[ $CustomKernel =~ "http" ]];then
+        install_package wget
+        IFS=', ' read -r -a array <<<  "$CustomKernel"
+        for element in "${array[@]}"; do
+            LogMsg "Web link detected. Downloading $element"
+            wget "$element"
+        done
+        if [[ "${#array[@]}" -gt 1 && "${array[0]}" = *.deb ]]; then
+            CustomKernel="*.deb"
+        fi
+        if [[ "${#array[@]}" -gt 1 && "${array[0]}" = *.rpm ]]; then
+            CustomKernel="*.rpm"
+        fi
+        if [[ "${#array[@]}" -eq 1 ]]; then
+            CustomKernel="${CustomKernel##*/}"
+        fi
+    fi
+
+    if [[ $CustomKernel = *tar.gz || $CustomKernel = *.tar ]]; then
+        tar -xf "${CustomKernel#$LOCAL_FILE_PREFIX}"
+        count=$(find -name "*.deb" -type f | wc -l)
```

Comment:
you can reuse this deb check also for the http part. just cycle trough the web files and download them or untar. then use this checks to see if it s rpm or deb. this way you remove 10 lines of code at least.

Reply:
Thanks, for tar ball, also support localfile:./script/rpm/msft419.tar.gz this way, let me think about it.